src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/DataSetSitemapItemViewModel.cs

```diff
@@ -0,0 +1,8 @@
+namespace GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
+
+public record DataSetSitemapItemViewModel
+{
+    public required string Id { get; init; }
+
+    public DateTime? LastModified { get; init; }
```

Comment:
Should these dates be _non_-nullable? Although a modification date would typically be nullable for most entities, `lastmod` on a sitemap item would always be present because presumably the initial value is when it was first created.